planex/link.py

```diff
@@ -56,7 +56,7 @@ def sources(self):
         if self.schema_version < 2:
             return self.link.get('sources', None)
 
-        patch_matcher = re.compile(r'source\d*', re.IGNORECASE)
+        patch_matcher = re.compile(r'\Asource\d*\Z', re.IGNORECASE)
```

Comment:
`match()` already implies start of line.

Reply:
Is `\Z` any clearer than `$`? (I had to look it up)

Reply:
No, I was not sure if python re supported `$` and I remembered that it did support `\A` and `\Z`. I'm happy to change it.

Reply:
Ops, I thought it was in my recently-merged PR. This is uniform with what is done in `spec.py`, if we change it here we should probably update it also there.